pkg/common/config/config.go

```diff
@@ -401,6 +399,10 @@ func validateConfig(cfg *Config) error {
 					return ErrPasswordMissing
 				}
 			}
+		} else if cfg.IsSecretInfoProvided() && !vcConfig.IsSecretInfoProvided() {
+			vcConfig.SecretHolder = DefaultCredentialManager
+		} else if vcConfig.IsSecretInfoProvided() {
+			vcConfig.SecretHolder = vcConfig.SecretName + "/" + vcConfig.SecretNamespace
```

Comment:
oops I think earlier I meant that this should be `namespace/name` which is consistent how resources are key'd in client-go

Reply:
Though I see now it doesn't really matter like you mentioend

Reply:
Yea, not a problem.